internal/cli/daemon/daemon.go

```diff
@@ -66,6 +66,17 @@ func NewCommand(srv rpc.ArduinoCoreServiceServer, settings *rpc.Configuration) *
 			if maxGRPCRecvMsgSize < 1024 {
 				feedback.Fatal(i18n.Tr("%s must be >= 1024", "--max-grpc-recv-message-size"), feedback.ErrBadArgument)
 			}
+
+			// The user agent should include "daemon" for analytics purposes
+			_, err := srv.SettingsSetValue(cmd.Context(), &rpc.SettingsSetValueRequest{
+				Key:          "network.user_agent_ext",
+				ValueFormat:  "cli",
+				EncodedValue: "daemon",
+			})
```

Comment:
What should we do in case this value is not empty? 
1. Should we always override with the `daemon` value
2. should we append to the previous value
3. should we honor the user setting?

/cc @cmaglie